R/pkg/R/functions.R

```diff
@@ -1179,6 +1179,27 @@ setMethod("soundex",
             column(jc)
           })
 
+#' Return the partition ID as a column
+#'
+#' Return the column for partition ID of the Spark task.
```

Comment:
Couple of minor nits: 
- To be consistent with the title this line can be `Return the partition ID of the Spark task a SparkDataFrame column`
- I think `nondeterministic` is more suitable than `indeterministic`